controller/backup_volume_controller.go

```diff
@@ -212,21 +212,18 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 
 	log := getLoggerForBackupVolume(bvc.logger, backupVolume)
 
-	// Get default backup target
-	backupTarget, err := bvc.ds.GetBackupTargetRO(types.DefaultBackupTargetName)
-	if err != nil {
-		if apierrors.IsNotFound(err) {
-			return nil
-		}
-		return errors.Wrapf(err, "failed to get %s backup target", types.DefaultBackupTargetName)
+	// Get the backup target of the backup volume
+	backupTarget, err := bvc.ds.GetBackupTargetRO(backupVolume.Spec.BackupTargetName)
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Why skip the case `apierrors.IsNotFound(err)`? Should we return `nil` as the previous implementation?

Reply:
Skip it because the backup target might be deleted before the backup volumes.
And We only have one backup target in the current implementation, which means the backup target is deleted only when uninstallation. The backup volumes and backups will be deleted during the uninstallation.
Need to handle the backup target is deleted before backup volumes but we add the owerRerernce into the backup volume CR, this should not happen, just in case.